exampleCourse/questions/examplesNumberInput/question.html

```diff
@@ -0,0 +1,81 @@
+<div class="card my-2">
+  <div class="card-body">
+    Questions below were designed to showcase the different features of 
+    <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a>. 
+    The goal of <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a>
+    is to allow users to enter <b>real or complex values</b> to be checked within a specific tolerance.
+  </div>
+
+  <div class="card-body">
+    <pl-question-panel>
+      <p>Within this question, you can observe the default behavior of the <a href="http://prairielearn.readthedocs.io/en/latest/elements/#pl-number-input-element"><code>pl-number-input</code></a> element.
+        In particular, the default input field has help text as a placeholder as well as a question button and is set to use <code>"relabs"</code> to check the tolerance, which is a combination of relative (<code>"rtol"</code>) and absolute (<code>"atol"</code>) tolerances.</p>
+      <p> Solve for the value of $c$ given $c^2 = {{params.a}}^2 + {{params.b}}^2$ </p>
```

Comment:
Might be nice to mention the solution to this equation at the very start of this question so that lazy people (like me) can quickly get a sense of what the right answer is so they can observe how the input behaves with a correct answer.

Reply:
I can modify the sample question to be something easier (sorry, I thought the above was simple enough). What about ask to enter the dimensions of a matrix?

Reply:
It's not too hard as is! I was just like, ever so slightly too lazy to actually get out my calculator :) just mentioning at the top of the question "if you don't feel like getting out your calculator, the correct answer is 10.xxxxxx" would be enough for me